app/src/components/common/Pagination.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+  <div class="pagination">
+    <NumberInput
+      id="number-input"
+      label="Rows per page"
+      :min="1"
+      :max="totalRows"
+      :value="rowsPerPage"
+      type="number"
+      @input="updateRowsPerPage"
+    />
+    <div class="pages">
+      <Button
+        :style-type="currentPage === 1 ? 'disabled' : 'secondary'"
```

Comment:
Let's move this condition into getter.

Reply:
Moved condition to the getters in the recent update